package.json

```diff
@@ -11,12 +11,13 @@
   "scripts": {
     "lint": "gts check",
     "lint:fix": "gts fix",
-    "start": "ts-node index.ts",
-    "test:build": "tsc --noEmit",
+    "start": "npm run compile:spectypes && ts-node index.ts",
+    "test:build": "npm run compile:spectypes && tsc --noEmit",
     "test": "jest",
```

Comment:
Before starting jest, shouldn't there be a `npm run compile:spectypes` compile step too?

The `test:build` script is a bit misleading :sweat_smile:  - it is actually just running Typescript type checks and not "building" anything.

Reply:
fixed